homeassistant/components/trafikverket_weatherstation/sensor.py

```diff
@@ -144,18 +152,39 @@ class TrafikverketSensorEntityDescription(
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
-    """Set up the Trafikverket sensor platform."""
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType = None,
+) -> None:
+    """Import Trafikverket Weather configuration from YAML."""
+    _LOGGER.warning(
```

Comment:
```suggestion
    # Config flow added in Home Assistant Core 2021.12, remove import flow in 2022.4
    _LOGGER.warning(
```

Reply:
Somehow I missed this one in fixing the sensor so it's now added back